tests/helix/send-to-helix-buildonhelixtests.targets

```diff
@@ -35,7 +35,7 @@
       <_TestBlameArguments Include="--blame-crash-dump-type full" />
 
       <!-- Using `dotnet test` for the project directly here -->
-      <_TestRunCommandArguments Include="dotnet test -s .runsettings --results-directory $(_HelixLogsPath)" />
+      <_TestRunCommandArguments Include="dotnet test -s .runsettings --results-directory $(_HelixLogsPath) -v:n" />
```

Comment:
Do we need this when the .runsettings also has:

```
      <Logger friendlyName="console">
        <Configuration>
          <Verbosity>normal</Verbosity>
        </Configuration>
      </Logger>
```

?

Reply:
Passing `-v:n` to `dotnet test` changes the verbosity of the MSBuild operations that `dotnet test` does, which is different than the verbosity of the console logger.

Reply:
We need `-v:n` so the msbuild logger shows the live output.